### DISCUSSION TITLE
MPV full screen causes Gnome to crash

### DISCUSSION THREAD

**Discussion by Neverminder** [smplayer-dev/smplayer#149] [Q&A]
So I'm on Ubuntu 20.04 and I'm using Smplayer with mpv, because with mplayer it doesn't work at all. Going to full screen causes Gnome to crash. I found the issue, it's this one: https://gitlab.gnome.org/GNOME/gnome-shell/-/issues/1678

If I launch mpv separately with the option "--x11-bypass-compositor=never" mentioned in the above link, the issue doesn't come up, but when I put the same option in Smplayer's "Here you can pass options and filters to MPlayer/mpv" field, it is ignored, unless "Run MPlayer/mpv in it's own window" is checked, which is not what I want.

Is there a way to make this work?

**Comment by smplayer-dev** [comment]
I guess that option doesn't work when the video is embedded in another application.

You can try running the new appimage as explained [in this post](https://github.com/smplayer-dev/smplayer/issues/6#issuecomment-878352302), and configured to use the `shared memory` video ouput. That way the video is rendered by smplayer, not by mpv. Maybe that would prevent that gnome crash.

**Reply by Neverminder** [reply]
This is just for Wayland though, isn't it? Because I'm running Ubuntu 20.04 which is still X11.
Another observation on the same subject - selecting mplayer causes the same Gnome crash when switching to full screen. Surely I can't be the only one affected by this issue?

**Reply by smplayer-dev** [reply]
It's not only for Wayland. It works for X11 as well.

**Reply by Neverminder** [reply]
> I guess that option doesn't work when the video is embedded in another application.
> 
> You can try running the new appimage as explained [in this post](https://github.com/smplayer-dev/smplayer/issues/6#issuecomment-878352302), and configured to use the `shared memory` video ouput. That way the video is rendered by smplayer, not by mpv. Maybe that would prevent that gnome crash.

So I tried the AppImage solution, but it didn't change anything. Gnome is still crashing.
